backend/src/packages/projects/project.service.ts

```diff
@@ -23,6 +25,24 @@ class ProjectService implements Omit<IService, 'find' | 'update' | 'delete'> {
     };
   }
 
+  public async findByUserId({
+    userId,
+  }: FindProjectByUserIdRequest): Promise<ProjectGetAllResponseDto> {
+    // const items = await this.projectRepository.findByUserId(id);
+
+    const items = await this.projectRepository.findByUserId(userId);
+
+    if (!items) {
+      throw new ApplicationError({
+        message: 'This User Has No Projects',
```

Comment:
I think you should just return empty array in this case

Reply:
\+  and also this check has no sense as `items` is array and empty array is truthy value